docs/reference/mpremote.rst

```diff
@@ -0,0 +1,126 @@
+.. _mpremote_files:
+
+MicroPython .mpremote files
+===========================
+
+This CLI tool provides an integrated set of utilities to remotely interact with and automate a MicroPython device over a serial connection.
+
+The simplest way to use this tool is::
+
+mpremote
+
+This will automatically connect to the device and provide an interactive REPL.
+
+The full list of supported commands are:
+
+*  connect to the given device
+
+  ..code-block:: python3
+  
+    $ mpremote <device-shortcut>       
+* connect to given device (device may be: list, auto, id:x, port:x
```

Comment:
Please put a blank line before each bullet point to make it easier to read this rst file (unless that breaks the output formatting...).